README.md

```diff
@@ -1,9 +1,27 @@
 # Pillar Prototype
 These salt files are intended to allow the creation and management of multiple ceph clusters with a single salt master.  
 
-The diagram should explain the intended flow for the orchestration runners and related salt states.  
+The [diagram](pillar-proposal.png) should explain the intended flow for the orchestration runners and related salt states.
 
 ## Status
 This is alpha at best.  It works, but runners need refactoring, commenting, correct returns and unit tests.  Several decisions still remain and a few files have notes alluding to missing functionality.  Only a single cluster named `ceph` has been tested.
 
+## Usage
 
+- Install salt-master on one host
+- Install salt-minion on all the other hosts you want to be ceph nodes, and *also* on the master host
+- Install https://github.com/oms4suse/python-ceph-cfg on all hosts.
+- Copy etc/salt/master.d/reactor.conf-example to /etc/salt/master.d/reactor.conf on the master, or set "startup_states: 'highstate'" in all the minion configs.
+- Accept all the salt keys on the master, make sure salt works (`salt '*' test.ping`)
+- Copy the srv tree from this repo to /srv on the salt master (be careful if you're copying onto an existing salt master which already has top.sls files and whatnot).
+- Create an SLS file in /srv/pillar/ceph/cluster/ for each host, indicating what cluster the host belongs to.  For ceph nodes, the file should contain just the line "cluster: ceph".  For the admin node (or salt master), use "cluster: unassigned".
```

Comment:
This could be changed now to usage of 'salt-run bootstrap.all' since this runner was merged. This would make all salt minions to ceph cluster nodes. A more selective runner is on the way.